pkg/configuration/constants.go

```diff
@@ -4,7 +4,7 @@ const (
 	API_URL                         string = "snyk_api"                // AKA "endpoint" in the config file
 	AUTHENTICATION_SUBDOMAINS       string = "internal_auth_subdomain" // array of additional subdomains to add authentication for
 	AUTHENTICATION_TOKEN            string = "snyk_token"
-	AUTHENTICATION_BEARER_TOKEN     string = "bearer_token"
+	AUTHENTICATION_BEARER_TOKEN     string = "snyk_bearer_token"
```

Comment:
Suggestion: To reduce lookups, better use the actual variable name 'snyk_oauth_token', which can then be removed from cli main

Reply:
updated 👍